CHANGELOG.md

```diff
@@ -17,12 +17,15 @@ Go to [legend](#legend---types-of-changes) for further information about the typ
 	- bug_report.md and feature_request.md
 	- CODE_OF_CONDUCT.md file
 	- PULL_REQUEST_TEMPLATE.md file
+- _WD_JsonCookie() (mLipok)
 
 ### Changed
 
 - README.md restruturing
```

Comment:
restructuring ? typo?

Reply:
Must have been from when I "fixed" the conflicts. I will correct it.

Reply:
OIC.... You meant the parentheses. 🙄 I'll fix it.

Reply:
no parentheses.

`- README.md restruturing` >> `- README.md restructuring` there is missing `c`

Reply:
> restruturing 

Ah... ok. Wasn't clear what you meant. I saw that line 20 was highlighted and focused only on that.

Reply:
Line 20 was changed and for this reason is marked on green.
It is not focused line.
My review was focused on line 24, and this is something you have to get used to on github.